src/ol/expr/expression.js

```diff
@@ -102,6 +102,7 @@ import {isStringColor} from '../color.js';
  *     (e.g. `'#86A136'`), colors using the rgba[a] functional notation (e.g. `'rgb(134, 161, 54)'` or `'rgba(134, 161, 54, 1)'`),
  *     named colors (e.g. `'red'`), or array literals with 3 ([r, g, b]) or 4 ([r, g, b, a]) values (with r, g, and b
  *     in the 0-255 range and a in the 0-1 range) (WebGL only).
+ *   * `['to-number', value1]` converts `value1` to a `number`. (Flat only)
```

Comment:
I just realized that the `to-number` operator in Maplibre [takes multiple fallback values as well](https://maplibre.org/maplibre-style-spec/expressions/#types-to-number). This is what the documentation should look like:

```suggestion
 *   * `['to-number', value, fallback1, fallback2, ...]` converts `value` to a number. If the input is null or false, the result is 0. If the input is true, the result is 1. If the input is a string, it is converted to a number if possible. If multiple values are provided, each one is evaluated in order until the first successful conversion is obtained. If none of the inputs can be converted, an error is thrown. (Canvas only)
```